core/unit_test/CMakeLists.txt

```diff
@@ -104,6 +105,20 @@ if(WIN32)
   LIST(APPEND COMPILE_ONLY_SOURCES TestWindowsInclude.cpp)
 endif()
 
+if(Kokkos_INSTALL_TESTING)  # FIXME Kokkos_ and KOKKOS_ variables are out of sync
```

Comment:
Brief comment why there is some special case handling here (even just "See Kokkos_TypeInfo.hpp")

Reply:
https://github.com/kokkos/kokkos/issues/7305 is intended to keep track of that issue.
I was not planning on adding a link in the code, it feels unnecessary.